apps/storybook/src/stories/datacom-loader.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from 'react';
+import { StoryFn } from '@storybook/react';
+import { DatacomLoader, DatacomButton } from '@datacom/endeavour-react';
+import styled from '@emotion/styled';
+
+type LoaderProps = React.ComponentProps<typeof DatacomLoader>;
+
+export default {
+  title: 'Loader',
+  component: DatacomLoader,
+  argTypes: {
+    label: {
+      name: 'Label',
+      defaultValue: 'Loading message',
+      description: 'Label for loading status.',
+      type: { name: 'string' },
+    },
+    loadingStatus: {
+      name: 'Loading Status',
+      defaultValue: 'default',
+      control: 'select',
+      description: 'Loading status is default if it is not set',
+      options: ['default', 'none', 'error', 'success'],
+    },
+  },
+};
+
+const Template: StoryFn<LoaderProps> = (args) => {
+  return <DatacomLoader></DatacomLoader>;
+};
+
+export const InlineLoader = (args) => {
```

Comment:
args parameter has an error complaining about having an 'any' type (i.e. it's not typed at all.) You should give this a type of the Loader props plus the label.

Have a look at the ContentTag story file (line 43) for an idea of how that can be done.

Reply:
It still shows a typed error even though the assigned type.